operator/v1/types_ingress.go

```diff
@@ -1037,6 +1039,18 @@ type SyslogLoggingDestinationParameters struct {
 // ContainerLoggingDestinationParameters describes parameters for the Container
 // logging destination type.
 type ContainerLoggingDestinationParameters struct {
+	// maxLength is the maximum length of the log message.
+	//
+	// Valid values are integers in the range 480 to 8192, inclusive.
+	//
+	// When omitted, the default value is 1024.
+	//
+	// +kubebuilder:validation:Optional
```

Comment:
You don't need this line as you already have `+optional` below

Reply:
This comment was resolved but hasn't actually been resolved, please drop this line as you have `+optional` already